### PR TITLE
PEO-1147 Bump ruby to 2.7.1

### PR DESCRIPTION
**Pull request by WoLkErSs**
[PEO-1147](https://garage.atlassian.net/browse/PEO-1147)

### Description

Bump ruby version to 2.1.7
Bump bundler to 2.1.4

### Before submitting the merge request make sure the following are checked

- [x] Followed the style guidelines of this project
- [x] Performed a self-review of own code
- [x] Wrote the tests that prove that fix is effective/that feature works

### CODE REVIEW

File: lib/macro/model.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ module Macro
   def self.Model(entity:, connections: [], find_by_key: :id, params_key: :id, assign: false)
     task = ->((ctx, flow_options), **) {
       if assign
-        Macro::Assign(to: :model, path: [*entity, *connections], try: true)[:task].call(ctx)
+        Macro::Assign(to: :model, path: [*entity, *connections], try: true)[:task].call(ctx, **{ options: [] })
```
Comment (bestwebua) [inline lib/macro/model.rb:7]: Why are you update this args passing? Please describe.

File: spec/concepts/api/v1/users/lib/service/session_token_spec.rb [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe Api::V1::Users::Lib::Service::SessionToken do
   describe '.create' do
-    subject(:service) { described_class.create(params) }
+    subject(:service) { described_class.create(**params) }
```
Comment (bestwebua) [inline spec/concepts/api/v1/users/lib/service/session_token_spec.rb:5]: For what double splat??? It's do the same.

File: spec/concepts/api/v1/lib/service/json_api_spec.rb [hunk 3]
```diff
@@ -33,9 +33,9 @@ module Api::V1::Lib::Service::JsonApi # rubocop:disable Style/ClassAndModuleChil
     end
 
     context 'when required parameter not passed' do
-      let(:params) { {} }
+      let(:params) { { type: :string } }
```
Comment (bestwebua) [inline spec/concepts/api/v1/lib/service/json_api_spec.rb:36]: ???

**Review by bestwebua — CHANGES_REQUESTED** [review]
PR must be an corpuscular. Slice this pr to single topic pr's with details what you want to improve.

File: lib/macro/model.rb (hunk 1 above)
Comment (WoLkErSs) [reply lib/macro/model.rb:7]: [ruby-2.7](https://www.ruby-lang.org/en/news/2019/12/25/ruby-2-7-0-released/)

File: spec/concepts/api/v1/users/lib/service/session_token_spec.rb (hunk 2 above)
Comment (WoLkErSs) [reply spec/concepts/api/v1/users/lib/service/session_token_spec.rb:5]: same

File: spec/concepts/api/v1/lib/service/json_api_spec.rb (hunk 3 above)
Comment (WoLkErSs) [reply spec/concepts/api/v1/lib/service/json_api_spec.rb:36]: same

**Review by bestwebua — APPROVED** [review]

File: .circleci/config.yml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 defaults: &defaults
   working_directory: ~/boilerplate-rails-api
   docker:
-    - image: circleci/ruby:2.6.5-node
+    - image: circleci/ruby:2.7.1-node-browsers-legacy
```
Comment (chubchenko) [inline .circleci/config.yml]: Why do we need `legacy`?

**Comment by chubchenko** [comment]
@WoLkErSs could you please resolve conflicts and update circleci configuration (need to remove install of bundler)

**Pull request merged**